### PR TITLE
refactor(grpc): add set_grpc_meta

### PR DESCRIPTION
**Pull request by majorgreys**
Refactor span metadata code that was repeated in both the client and server interceptor.

### CODE REVIEW

File: ddtrace/contrib/grpc/client_interceptor.py [hunk 1]
```diff
@@ -177,17 +177,10 @@ def _intercept_client_call(self, method_kind, client_call_details):
...
-        # tags for method details
-        method_path = client_call_details.method
-        method_package, method_service, method_name = parse_method_path(method_path)
-        span._set_str_tag(constants.GRPC_METHOD_PATH_KEY, method_path)
-        span._set_str_tag(constants.GRPC_METHOD_PACKAGE_KEY, method_package)
-        span._set_str_tag(constants.GRPC_METHOD_SERVICE_KEY, method_service)
-        span._set_str_tag(constants.GRPC_METHOD_NAME_KEY, method_name)
-        span._set_str_tag(constants.GRPC_METHOD_KIND_KEY, method_kind)
+        set_grpc_method_meta(span, client_call_details.method, method_kind)
+        span._set_str_tag(constants.GRPC_SPAN_KIND_KEY, constants.GRPC_SPAN_KIND_VALUE_SERVER)
```
Comment (Yun-Kim) [inline ddtrace/contrib/grpc/client_interceptor.py:181]: Just double checking that the right constant is being used here?
```suggestion
        span._set_str_tag(constants.GRPC_SPAN_KIND_KEY, constants.GRPC_SPAN_KIND_VALUE_CLIENT)
```

**Review by Yun-Kim — COMMENTED** [review]
Lgtm otherwise a small detail

File: ddtrace/contrib/grpc/client_interceptor.py (hunk 1 above)
Comment (majorgreys) [reply ddtrace/contrib/grpc/client_interceptor.py]: Thanks for catching that!!!

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Comment by majorgreys** [comment]
@Mergifyio backport 0.48

**Pull request merged**